deps/src/main/java/com/microsoft/azure/sdk/iot/deps/auth/ProvisioningSasToken.java

```diff
@@ -22,6 +23,7 @@
 {
     private static final long TOKEN_VALID_SECS = 365*24*60*60;
     private static final long ONE_SECOND_IN_MILLISECONDS = 1000;
+    private static final long BUFFER_SECONDS = 0;
```

Comment:
Do we need this value if it is always 0? Or is this for configuring later?

Reply:
It's something we can configure later if required